script.js

```diff
@@ -1,4 +1,185 @@
+var gameMode_ROLL_DICE = "GAME_ROLL_DICE";
+var gameMode_CHOOSE_DICE = "CHOOSE_DICE";
+var gameMode_COMPARE_SCORES = "COMPARE_SCORES";
```

Comment:
I would suggest to stick to either all UPPERCASED or camelCase rather than a mix